community/easy-rsa/APKBUILD

```diff
@@ -6,7 +6,7 @@ pkgdesc="Simple shell based CA utility"
 url="https://github.com/OpenVPN/easy-rsa"
 arch="noarch"
 license="GPL-2.0"
-depends="openssl1.0"
+depends="libressl"
```

Comment:
that could wait til transition to `openssl`

Reply:
The transition to `opnessl` seems to block a lot of things here (e.g. #5406). Is there a timeline this?

Reply:
transition done!